auction-server/src/auction/service/verification.rs

```diff
@@ -578,25 +578,43 @@ impl Service<Svm> {
     }
 
     pub async fn simulate_bid(&self, bid: &entities::BidCreate<Svm>) -> Result<(), RestError> {
-        let response = self
-            .config
-            .chain_config
-            .simulator
-            .simulate_transaction(&bid.chain_data.transaction)
-            .await;
-        let result = response.map_err(|e| {
-            tracing::error!("Error while simulating bid: {:?}", e);
-            RestError::TemporarilyUnavailable
-        })?;
-        match result.value {
-            Err(err) => {
-                let msgs = err.meta.logs;
-                Err(RestError::SimulationError {
-                    result: Default::default(),
-                    reason: msgs.join("\n"),
-                })
-            }
-            Ok(_) => Ok(()),
+        const RETRY_LIMIT: usize = 5;
+        let mut retry_count = 0;
+        loop {
+            let response = self
+                .config
+                .chain_config
+                .simulator
+                .simulate_transaction(&bid.chain_data.transaction)
```

Comment:
i think the retry should happen within the fetching? before the simulator actually runs? otherwise it's a weird interface where we're asking for a minimum slot to simulate against and then ignoring that as long as it passes simulation.

Reply:
we can also add some checks to make sure searchers aren't submitting slots that are more than some reasonable number off of where the rpc currently stands

Reply:
I consider this some internal behaviour which is subject to change. Interface only adds the option for a searcher to guarantee successful simulation from the specified slot. How we verify this internally is not part of the interface.

I classify that as a DoS attack and don't think it's a priority

Reply:
in that case, maybe it's worth a comment baking that into the interface on the SDK side, or in the `BidChainDataCreateSvm` object in `auction-server/src/auction/entities/bid.rs`

Reply:
there is a comment already there. do you think we should explain more?